### ISSUE TITLE
Blood Cell Detection using DL

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#238]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Blood Cell Detection</br>
:red_circle: **Aim** : Create a model which will detect the blood cells.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/adhoppin/blood-cell-detection-datatset </br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by rafiya2003** [comment]
I would like to work on this, frankly I don't know much but I will learn and do. I am a newbie(first year UG student). I want to learn and grow

**Comment by Abhinavcode13** [comment]
Please assign this issue to me @abhisheks008

**Comment by abhisheks008** [comment]
Please do not to spam in every single issue. This might cause escalation as you are breaching Code of Conduct and Contribution Guidelines.

@rafiya2003

**Comment by Sagar201003** [comment]
I can work on this project . Believe me , I will try my 100% . 

FullName: Sagar Santoshkumar Shukla
Github Profile Link: https://github.com/Sagar201003
Email ID: shuklasagar201003@gmail.com

Approach for this Project:
I will perform EDA , feature selection , PCA , Data augmentation . I will use Pycaret to get the best model .
Implement the models using TensorFlow or PyTorch.

What is your participant role? -SSOC contributor

**Comment by jatinsahani** [comment]
Full name : Jatin Sahani
GitHub Profile Link : https://github.com/jatinsahani
Email ID : sahanijatin00@gmail.com
Participant ID (if applicable): 
Approach for this Project :

There can Three approaches for the detection of Blood Cell; that are:  
-Region Based Convolutional Neural Network 
- VGG-16
-YOLO

As YOLO is 3x faster than then R-CNN, I guess using it will be one of the best move. We can also note that the dataset provided is already Processed for YOLO.


*I am new to this field but I have bookish Knowledge, My practical knowledge is null but I would love to do this Project*
*Hope to learn and grow*



What is your participant role? SSOC Contributor

**Comment by abhisheks008** [comment]
Issue assigned to you @jatinsahani

**Comment by jatinsahani** [comment]
> Issue assigned to you @jatinsahani 

Thank You

**Comment by jatinsahani** [comment]
Acknowledged

**Comment by MattiaFerrarini** [comment]
Dear Abhishek, unfortunately I am not able to keep working on the issue because of academic duties. You can assign it to someone else. I'm sorry for wasting your time.

**Comment by abhisheks008** [comment]
No issues @MattiaFerrarini

**Comment by Sneha-Mahata** [comment]
Hello, I would like to work on this task. The details hereby:

Full name : Sneha Mahata
GitHub Profile Link : https://github.com/Sneha-Mahata
Email ID : [mahatasneha4@gmail.com](mailto:mahatasneha4@gmail.com)
Participant ID (if applicable): NA
Approach for this Project : I plan to approach this project by conducting an exploratory data analysis, implementing multiple object detection models such as YOLO, Histogram of Oriented Gradients (HOG), Single Shot Detector (SSD), Faster R-CNN etc. I will tune the hyperparameters, employ data augmentation techniques, and evaluate the models using appropriate metrics. After comparing the accuracies among these models, I will select the best-suited one as my final model. The entire process will be documented in the README.md file.
What is your participant role?  GSSOC 2024 Contributor

**Comment by abhisheks008** [comment]
Issue assigned to you @Sneha-Mahata. You can start working on it.

**Comment by Sneha-Mahata** [comment]
> Issue assigned to you @Sneha-Mahata. You can start working on it.

Thanks a lot, I'm excited to work on it !

**Comment by Akshitha2912v** [comment]
Hi,I would like to work on this issue under GSSoC 2026.
Could you please assign this issue to me?
Thank you!
